hl7v23/datatype_cm_pta.bal

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein is strictly forbidden, unless permitted by WSO2 in accordance with
+// the WSO2 Software License available at: https://wso2.com/licenses/eula/3.2
+// For specific language governing the permissions and limitations under
+// this license, please see the license as well as any agreement you’ve
+// entered into with WSO2 governing the purchase of this software and any
+// associated services.
+
```

Comment:
Some files have old license headers.

Reply:
Please find the fixes with 1ee5decbe5c4a2571579ee34c3eddd8ceeb0817e